apps/src/lib/client/tx.rs

```diff
@@ -459,17 +459,14 @@ pub async fn submit_change_consensus_key(
                 .save()
                 .unwrap_or_else(|err| edisplay_line!(namada.io(), "{}", err));
 
-            // let tendermint_home = config.ledger.cometbft_dir();
-            // tendermint_node::write_validator_key(
-            //     &tendermint_home,
-            //     &consensus_key,
-            // );
-            // tendermint_node::write_validator_state(tendermint_home);
-
             display_line!(
                 namada.io(),
-                "  Consensus key \"{}\"",
-                consensus_key_alias
+                "New consensus key stored with alias \
+                 \"{consensus_key_alias}\". It will become active \
+                 {EPOCH_SWITCH_BLOCKS_DELAY} blocks before pipeline offset \
+                 from the current epoch, at which point you'll need to give \
+                 the new key to CometBFT in order to be able to sign with it \
+                 in consensus.",
```

Comment:
can we somehow make this message dependent on if the transaction is valid or not? Playing around and testing to make sure if validator-1 submits the consensus key change for validator-0, then the tx is invalid. However, we still currently print this message when it is invalid.

Reply:
ah good catch! Looks like we need to do this for couple other txs too, so I'll do it in a follow up PR - the client output could be overall improved

Reply:
also noticed that for some reason, the storage keys for this inner tx show up in the client result as empty (they don't show in `Transaction applied with result:` info when they're empty) - I can see them populated in the node, so I'll open another issue to find why

Reply:
Opened #2267 for this